ext/afform/core/Civi/Api4/Action/Afform/AbstractProcessor.php

```diff
@@ -300,4 +334,118 @@ public function setName(string $name) {
     return $this;
   }
 
+  /**
+   * Replace Entity reference fields with the id of the referenced entity.
+   * @param string $entityName
+   * @param $records
+   */
+  public function replaceReferences($entityName, $records) {
```

Comment:
This function was previously `private` and I don't see you calling it outside the class so I think it can be `protected`

Reply:
Done